docs/quickstart.md

```diff
@@ -0,0 +1,157 @@
+# Quickstart Guide
+
+This guide helps you quickly get started with PMM using Docker.
```

Comment:
Rephrase:
The procedures on this page are designed to guide you through the simplest way to get up and running with Percona Monitoring and Management.

Reply:
Fixed